package.json

```diff
@@ -18,6 +18,7 @@
   "author": "David Boyne",
   "license": "MIT",
   "dependencies": {
+    "aws-sdk": "^2.896.0",
```

Comment:
perhaps use the new modularized and more lightweight v3?

Reply:
Didn't even know it existed! Will update

Reply:
done!

Reply:
Awesome!!